modules/Makefile

```diff
@@ -187,7 +193,7 @@ INTERNAL_MODULES_TO_DOCUMENT =                        \
 documentation: $(SYS_CTYPES_MODULE_DOC)
 	@echo "Generating module documentation with chpldoc"
 	@export CHPLDOC_AUTHOR='Hewlett Packard Enterprise Development LP' && \
-	  $(CHPLDOC) --save-sphinx ${MODULE_SPHINX} --no-html $(MODULES_TO_DOCUMENT) $(DISTS_TO_DOCUMENT) $(PACKAGES_TO_DOCUMENT) $(INTERNAL_MODULES_TO_DOCUMENT) $(SYS_CTYPES_MODULE_DOC)
+	  $(CHPLDOC) --home $(CHPL_HOME) --save-sphinx ${MODULE_SPHINX} --no-html $(MODULES_TO_DOCUMENT) $(DISTS_TO_DOCUMENT) $(PACKAGES_TO_DOCUMENT) $(INTERNAL_MODULES_TO_DOCUMENT) $(SYS_CTYPES_MODULE_DOC)
```

Comment:
Shouldn't this be CHPL_MAKE_HOME ?

Reply:
Sigh, yes...  Strangely, this time I did check it on my symbolic link and  it seemed to pass.  I must've gotten something wrong (possibly using muscle memory to jump back to my main dev directory at some point).  I'm having a disappointingly sloppy week.  :(